apps/judicial-system/web/src/routes/Admin/UserForm/UserForm.tsx

```diff
@@ -218,139 +217,232 @@ export const UserForm: FC<Props> = ({
+                <RadioButton
+                  name="role"
+                  id="roleProsecutorRepresentative"
+                  label={userRoleToString(UserRole.PROSECUTOR_REPRESENTATIVE)}
+                  checked={user.role === UserRole.PROSECUTOR_REPRESENTATIVE}
+                  onChange={() =>
+                    setUser({
+                      ...user,
+                      role: UserRole.PROSECUTOR_REPRESENTATIVE,
+                    })
+                  }
+                  large
+                />
+              </Box>
             </Box>
-            <Box className={styles.roleColumn}>
-              <RadioButton
-                name="role"
-                id="roleProsecutorRepresentative"
-                label="Fulltrúi"
-                checked={user.role === UserRole.PROSECUTOR_REPRESENTATIVE}
-                onChange={() =>
-                  setUser({ ...user, role: UserRole.PROSECUTOR_REPRESENTATIVE })
-                }
-                large
-              />
+            <Box className={styles.roleRow}>
+              {user.institution.id ===
+              '8f9e2f6d-6a00-4a5e-b39b-95fd110d762e' ? (
+                <>
+                  <Box className={styles.roleColumn}>
+                    <RadioButton
+                      name="role"
+                      id="rolePublicProsecutorStaff"
+                      label={userRoleToString(UserRole.PUBLIC_PROSECUTOR_STAFF)}
+                      checked={user.role === UserRole.PUBLIC_PROSECUTOR_STAFF}
+                      onChange={() =>
+                        setUser({
+                          ...user,
+                          role: UserRole.PUBLIC_PROSECUTOR_STAFF,
+                        })
+                      }
+                      large
+                    />
+                  </Box>
+                  <Box className={styles.roleColumn}>
+                    <RadioButton
+                      name="role"
+                      id="roleLocalAdmin"
+                      label={userRoleToString(UserRole.LOCAL_ADMIN)}
+                      checked={user.role === UserRole.LOCAL_ADMIN}
+                      onChange={() =>
+                        setUser({ ...user, role: UserRole.LOCAL_ADMIN })
+                      }
+                      large
+                    />
+                  </Box>
+                </>
+              ) : (
+                <>
+                  <Box className={styles.roleColumn}>
+                    <RadioButton
+                      name="role"
+                      id="roleLocalAdmin"
+                      label={userRoleToString(UserRole.LOCAL_ADMIN)}
+                      checked={user.role === UserRole.LOCAL_ADMIN}
+                      onChange={() =>
+                        setUser({ ...user, role: UserRole.LOCAL_ADMIN })
+                      }
+                      large
+                    />
+                  </Box>
+                  <Box className={styles.roleColumn} />
```

Comment:
Is this empty div needed for something?

Reply:
Just making sure all role radio buttons have the same size. This was an easy way to do that.